diff --git a/External/p7zip/C/7zTypes.h b/External/p7zip/C/7zTypes.h
index 778413e..52ffa6b 100644
--- a/External/p7zip/C/7zTypes.h
+++ b/External/p7zip/C/7zTypes.h
@@ -57,12 +57,12 @@ typedef unsigned char Byte;
 typedef short Int16;
 typedef unsigned short UInt16;
 
-#ifdef _LZMA_UINT32_IS_ULONG
-typedef long Int32;
-typedef unsigned long UInt32;
+#if __LP64__
+typedef unsigned int                    UInt32;
+typedef signed int                      Int32;
 #else
-typedef int Int32;
-typedef unsigned int UInt32;
+typedef unsigned long                   UInt32;
+typedef signed long                     Int32;
 #endif
 
 #ifdef _SZ_NO_INT_64
